Documentation/Blazorise.Docs/Pages/News/2023-11-20-release-notes-140.razor

```diff
@@ -0,0 +1,73 @@
+﻿@page "/news/release-notes/140"
```

Comment:
Isn't there already a release notes file in master, can you merge both versions?

Reply:
Don't worry about this. We both created it at the same time. I will merge them at some point.